woocommerce/woocommerce-android
# Analytics Performance: add pull-to-refresh tracks to analytics hub

Summary
==========
Fix  by creating a track event that will register every time a pull-to-refresh is triggered inside the Analytics Hub. This is necessary since PR #9363 will introduce a force refresh action every time the pull-to-refresh happens, overriding our cache system and skipping the performance strategy we're introducing.

How to Test
==========
1. Open the app and enter the Analytics Hub
2. Once the data is loaded, exit and reenter the Hub with the same range selection, and verify that no network operation is triggered.
3. Trigger a pull-to-refresh action. Now verify that a new request was made fetching the stats data inside the Hub again for that specific selection range, and the new track event is triggered as expected.

Update release notes:
